### PR TITLE
Add support for mpls files that have mutiple playitems

### PR DESCRIPTION
**Pull request by tleydxdy**

### CODE REVIEW

**Comment by tleydxdy** [comment]
wait, found some bug

**Comment by tleydxdy** [comment]
should be fixed

File: README [hunk 1]
```diff
@@ -8,17 +8,19 @@ bdchapters is a basic tool to generate XML based chapter format from mpls files
 
 ### Usage ##
 
-bdchapters -i <mpls file> -o <mkv xml chapter file>
+bdchapters -i <mpls file> -o <directory for mkv xml chapter file>
```
Comment (Booloki) [inline README]: Hi @tleydxdy ,
New `-s ` argument is missing in usage command and a small explanation would be nice.

**Comment by Booloki** [comment]
Thank you !

**Pull request merged**